code/modules/jobs/job_types/bl_fields.dm

```diff
@@ -6,20 +6,103 @@
 
 	access = list(ACCESS_DFS)
 	minimal_access = list(ACCESS_DFS)
-	forbids = "The NCR forbids: Chem and drug use such as jet or alcohol while on duty. Execution of unarmed or otherwise subdued targets without authorisation."
-	enforces = "The NCR expects: Obeying the lawful orders of superiors. Proper treatment of prisoners.  Good conduct within the Republic's laws. Wearing the uniform."
-	objectivesList = list("Leadership recommends the following goal for this week: Establish an outpost at the radio tower","Leadership recommends the following goal for this week: Neutralize and capture dangerous criminals", "Leadership recommends the following goal for this week: Free slaves and establish good relations with unaligned individuals.")
+
+	forbids = "The DFS forbids: Provoking wars. You're tough, but not undefeatable. Know what battles you can and can't afford. <br> \
```

Comment:
This is new-player info, please write out the acronym so it's clearer (applies to below cases too)